libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/DirectionsRouteDiffProviderTest.kt

```diff
@@ -19,19 +19,19 @@ class DirectionsRouteDiffProviderTest {
     @Test
     fun buildRouteDiffs() {
         val oldRoute = createTestNavigationRoute(
-            createTestLeg(57.14, 8.571, 42.85, 90, "low", 71),
-            createTestLeg(142.8, 28.57, 71.42, 120, "unknown", 42),
-            createTestLeg(85.71, 42.85, 57.14, 90, "unknown", 14),
-            createTestLeg(71.42, 14.28, 85.71, 90, "low", 57),
-            createTestLeg(42.85, 57.14, 28.57, 120, "low", 28),
+            createTestLeg(57.14, 8.571, 42.85, 90, "low", 71, 90),
```

Comment:
I do not test waypoints logs here because it will change after https://github.com/mapbox/mapbox-navigation-android/pull/6555 and will be easier to test then.